common/lib/api-client/models/location.js

```diff
@@ -10,6 +10,10 @@ module.exports = {
       jsonApi: 'hasMany',
       type: 'suppliers',
     },
+    category: {
```

Comment:
`locations_free_spaces` returns a location object, so the category needs to be added onto the `location` model.